.editorconfig

```diff
@@ -0,0 +1,11 @@
+# https://editorconfig.org
```

Comment:
I don't think adding a config in this PR makes sense. I'd create a new PR with just the config. This increases the chances of the PR being merged 😉